Formula/llvm-bolt.rb

```diff
@@ -0,0 +1,62 @@
+class LlvmBolt < Formula
+  desc "Post-link optimizer developed to speed up large applications"
+  homepage "https://llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.6/llvm-project-14.0.6.src.tar.xz"
+  sha256 "8b3cfd7bc695bd6cea0f37f53f0981f34f87496e79e2529874fd03a2f9dd3a8a"
+  # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+  head "https://github.com/llvm/llvm-project.git", branch: "main"
+  livecheck do
+    url :stable
+    regex(/^llvmorg[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "pkg-config" => :build
+  depends_on :linux
+  depends_on "llvm"
+  depends_on "python@3.10"
+
+  fails_with gcc: "5"
+
+  def install
+    args = %w[
+      -DLLVM_TARGETS_TO_BUILD=X86
+      -DCMAKE_BUILD_TYPE=Release
+      -DLLVM_ENABLE_ASSERTIONS=ON
```

Comment:
I don't think it is actually required to build LLVM with assertions in order to build Bolt.

You should be able to build Bolt from a non-assertion build of LLVM, you just shouldn't optimise your (non-assertion) LLVM build using Bolt. See https://github.com/llvm/llvm-project/issues/55004.

Reply:
I just enabled edits by maintainers and you can apply changes.